Dockerfile

```diff
@@ -1,49 +1,50 @@
-FROM	debian:wheezy
-
+FROM	   node:6.7.0-slim
```

Comment:
I think opers would know best... I'm not aware of anything that in wheezy base that we need though.